### PR TITLE
TabView: remove shadowcaster when using updated style

### PR DESCRIPTION
**Pull request by beervoley**
## Description

This PR removes the shadowCaster and shadowReceiver when using updated style.
## Motivation and Context

Because we don't need the shadow with the updated TabView I removed both ShadowCaster and ShadowReceiver
## How Has This Been Tested?

Manually.

### CODE REVIEW

File: dev/TabView/TabView.cpp [hunk 1]
```diff
@@ -139,7 +140,7 @@ void TabView::OnApplyTemplate()
 
     if (SharedHelpers::IsThemeShadowAvailable())
     {
-        if (!SharedHelpers::Is21H1OrHigher())
+        if (!SharedHelpers::Is21H1OrHigher() && !XamlControlsResources::IsUsingControlsResourcesVersion2())
```
Comment (StephenLPeters) [inline dev/TabView/TabView.cpp:143]: This whole change is a bit weird... we are collapsing the shadow caster by default when using the V2 style, but when using the V2 style we will never enter this block, so it doesn't look like we ever make it visible. So then, why does it exist?

Also I don't think I like the idea of our code changing behavior based on the XCR's style version api. why did you need to do this?

**Review by StephenLPeters — CHANGES_REQUESTED** [review]

File: dev/TabView/TabView.cpp (hunk 1 above)
Comment (beervoley) [reply dev/TabView/TabView.cpp]: @StephenLPeters you're absolutely right, I'm confused why I did it this way.
fixed now

Comment (beervoley) [reply dev/TabView/TabView.cpp]: @StephenLPeters you're absolutely right, I'm also confused as to why I did it this way. It's clearly incorrect.
Fixed now :)

**Review by leonMSFT — APPROVED** [review]

**Review by StephenLPeters — CHANGES_REQUESTED** [review]

File: dev/TabView/TabView.cpp [hunk 2]
```diff
@@ -90,7 +90,10 @@ void TabView::OnApplyTemplate()
 
     if (!SharedHelpers::Is21H1OrHigher())
     {
-        m_shadowReceiver.set(GetTemplateChildT<winrt::Grid>(L"ShadowReceiver", controlProtected));
+        if (auto shadowReceiver = GetTemplateChildT<winrt::Grid>(L"ShadowReceiver", controlProtected))
```
[the remark this answers is not in the document]
Comment (beervoley) [reply dev/TabView/TabView.cpp]: you're correct

**Review by StephenLPeters — APPROVED** [review]
